src/Chapter3.hs

```diff
@@ -910,6 +1008,19 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold Int
+
+instance Append Gold where
+  append (Gold x) (Gold y) = Gold $ x + y
+
+instance Append [a] where
+  append x y = x ++ y
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
  append = (++)
```